docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -97,14 +103,32 @@ curl ---request GET 'http://localhost:8092/actuator/backup/backup1'
 ### Example response
 
 ```json
-{
-  "State": "COMPLETED"
-}
-```
+  {
+    "backupId": "backup1",
+    "failureReason": null,
```

Comment:
Do we also need an example request + response for `GET actuator/backup`? Because if I am not mistaken this is just example request + response for `GET actuator/backup/{backupId}`. Or?

Reply:
Hm, so the response would be basically the same object structure, just a list of these objects instead. I added line 127 to clarify this and felt that it would be too much to add an example, assuming it's enough for the user to understand the object structure of the items in the list. Wdyt?